lib/oc-tests/test_sharePermissions.py

```diff
@@ -341,6 +370,30 @@ def move_in_overwrite(self, expected_success = False):
         target_file = os.path.join(self.shared_dir, target_file)
         return self._move(source_file, target_file, expected_success)
 
+    def copy_in(self, expected_success = False):
+        test_file = self._make_test_file()
+        target_file = 'test_copy_in.dat'
+
+        # upload the test file outside the shared dir first
+        self.oc.put_file(target_file, test_file)
+
+        # then copy that one into the shared dir
+        source_file = target_file
+        target_file = os.path.join(self.shared_dir, source_file)
+        return self._copy(source_file, target_file, expected_success)
+
+    def copy_in_overwrite(self, expected_success = False):
```

Comment:
Any reason to have these functions duplicated (copy_in_overwrite and copy_in)? The only difference is the target file.